backend/src/main/java/ca/gov/dtsstn/cdcp/api/config/SpringDocConfig.java

```diff
@@ -33,9 +33,15 @@ public class SpringDocConfig {
 
 	static final Logger log = LoggerFactory.getLogger(SpringDocConfig.class);
 
-	@Autowired ApplicationProperties applicationProperties;
-
-	@Autowired GitProperties gitProperties;
+	private final ApplicationProperties applicationProperties;
+	private final GitProperties gitProperties;
+
+	@Autowired
```

Comment:
`@Autowired` is usually not needed on constructors these days. There are exceptions, but usually the application won't start when `@Autowired` is required.